### ISSUE TITLE
should refresh 

### ISSUE THREAD

**Issue by Burane** [Flyrell/axios-auth-refresh#186]
Hello, thanks for this good library.

In the doc we can see : 
```typescript
{
    shouldRefresh: (error) => error?.response?.data?.business_error_code === 100385
}
```
But this seems to don't exist anymore, how can I specify to refresh on custom logic inside the request response ?

**Comment by xiazhichen1** [comment]
It seems that #182 is merged on Mar.1 but not yet published (the latest version is from Feb).

**Comment by Flyrell** [comment]
Released in v3.3.1 today, thank you for catching that @xiazhichen1 .
